nautobot/ui/urls.py

```diff
@@ -0,0 +1,29 @@
+# UI-related API endpoints
```

Comment:
I don't think we should put this file here, seeing as `nautobot/ui` is not a Python package nor is it a Django application. Since the views/serializers are defined in `core.api` let's just put these url patterns for the UI endpoints in there as well.